firebase/firebase.js

```diff
@@ -199,4 +199,48 @@ export class FirebaseWrapper {
       console.log(error);
     }
   }
+
+  async UserAddEvent (user, eventId) {
+    try {
+      const ref = await this._firestore.collection("User").doc(user)
+      await ref.update({
+        events: firebase.firestore.FieldValue.arrayUnion(eventId)
+    });
+    } catch (error) {
+      console.log(error)
+    }
+  }
+
+  async UserDelEvent (user, eventId) {
+    try {
+      const ref = await this._firestore.collection("User").doc(user)
+      await ref.update({
+        events: firebase.firestore.FieldValue.arrayRemove(eventId)
+    });
+    } catch (error) {
+      console.log(error)
+    }
+  }
+
+  async addInterest (user, categoryCode) {
```

Comment:
Function to add interests to User's interest array (Majority of the code looks like the UserAddEvent function (which adds event to a user's events array, but we can refactor later).